cmd/klepto.go

```diff
@@ -39,6 +39,10 @@ func init() {
 	viper.BindPFlag("nRows", Klepto.PersistentFlags().Lookup("rows"))
 }
 
+func Execute() {
+	Klepto.Execute()
+}
```

Comment:
What's the reason for this function? It seems unnecessary

Reply:
Ah, you're right. Removed it.